curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-functional-programming-by-building-a-spreadsheet/661f48f412d7631a1d9c30e6.md

```diff
@@ -0,0 +1,140 @@
+---
+id: 661f48f412d7631a1d9c30e6
+title: Step 75
+challengeType: 0
+dashedName: step-75
+---
+
+# --description--
+
+You should use `console.log()` to return the `highPrecedence` function with the string `"5*3"`.
+
+# --hints--
+
+Please return the function using `console.log(highPrecedence("5*3"));`.
+
+```js
+
+assert.match(code, /console\.log\(highPrecedence\("5\*3"\)\);/);
+
+```
```

Comment:
We should make this more granular.

- Test for a function call of `highPrecedence`
- Test that `5*3` is passed as the argument
- Test that the result is logged.